common/providers/AccountProvider.js

```diff
@@ -88,6 +88,18 @@ const AccountProvider = class extends Component {
 
     render() {
         const { isLoading, isSaving, user, organisation, organisations, error } = this.state;
+        if (isLoading) {
+            return (
+                <div style={{
+                    position: 'absolute',
+                    top: '35%',
+                    width: '100%',
+                    textAlign: 'center',
+                }}>
```

Comment:
**Note**: Using inline-styles instead of defining these styles in a css/sass file & using a className.
This is because if we use css className approach, if there is a delay in loading the css styles, 
the `loader/spinner` is initially rendered at top-left of the screen and then eventually moves to the center of the page causing a flicker.

Better approach would have been to use styled-components, but since the entire app doesn't amke use of styled-components - using inline-styles in this case.

Reply:
That makes total sense, though actually I think the CSS only wouldn't load before the content during local development due to the way webpack extracts the css during production bundles?